README.md

```
@@ -235,13 +235,10 @@ You should add a reference to `opencvsharp/src/build/OpenCvSharpExtern/libOpenCv
 export LD_LIBRARY_PATH="${LD_LIBRARY_PATH}:/home/shimat/opencvsharp/src/build/OpenCvSharpExtern"
 ```
 
-- Add `OpenCvSharp4` NuGet package to your project
+- Build the managed OpenCvSharp library
 ```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)